website/docs/kubernetes/kind/deploying-an-image-to-kind.md

```diff
@@ -0,0 +1,28 @@
+---
+sidebar_position: 10
+title: Deploying an image to Kind
+description: Deploying an Podman image to your Kind cluster
+keywords: [podman desktop, podman, containers, images, migrating, kubernetes]
+tags: [migrating-to-kubernetes, images]
+---
+
+# Deploying an image to your local Kind-powered Kubernetes cluster 
+
+With Podman Desktop, you can deploy a Podman image to your local Kind-powered Kubernetes cluster.
+
+#### Prerequisites
+
+* Your are using the Podman container engine.
+* Your image is available on the **Images** page: *<your_image>*.
+* [You created a Kind cluster](creating-a-kind-cluster.md).
+
+#### Procedure
+
+1. Open **Podman Desktop dashboard > Images**.
+1. **<icon icon="fa-solid fa-search" size="lg" /> Search images**: `<your_image>`.
+1. Click **<icon icon="fa-solid fa-ellipsis-v" size="lg" /> > <icon icon="fa-solid fa-ellipsis-v" size="lg" /> Push image to Kind cluster**.
+1. Choose your kind cluster in the list.
+
+#### Verification
+
+* TODO
```

Comment:
should be removed of completed ?

Reply:
Completed. Yesterday my tests were not successful